dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -572,5 +573,82 @@ private void VerifyVerifyHeaderContentMargin(NavigationViewPaneDisplayMode paneD
                 root: headerContent,
                 masterFilePrefix: masterFilePrefix);
         }
+
+        [TestMethod]
+        public void VerifyMenuItemAndContainerMappingMenuItemsSource()
+        {
+            NavigationView navView = null;
+            ObservableCollection<String> menuItems = new ObservableCollection<String>();
+            menuItems.Add("Item 1");
+            menuItems.Add("Item 2");
+
+            RunOnUIThread.Execute(() =>
+            {
+                navView = new NavigationView();
```

Comment:
var navView = new NavigationView();
and you can remove the definition outside the block. You could also avoid the menuItems variable and just assign create and assign the collection directly.
navView.MenuItemsSource = new ObservableCollection<string> { "Item 1" , "Item 2" };